examples/integrated_gradients_imdb.ipynb

```diff
@@ -19,7 +19,7 @@
   },
```

Comment:
<p>Can you upgrade TF to 2.5.0, rerun the notebook and check if you get <code>tf.int32</code>  warnings when calling <code>explain</code>  please?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SeldonIO/alibi/pull/417/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>